.github/workflows/health_check.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: ['10', '12', '14']
+        node: ['12']
```

Comment:
not sure you want to do this if we care to support all versions. You can set `max-parallel: 1` to run these in series instead of parallel and then we'll only be alerted on the first node version test suite that fails. See https://github.com/patch-technology/patch-ruby/blob/4c8f0f5fd64a46f4e4815764be3ffb677be425db/.github/workflows/test.yml#L16

Reply:
Good idea. I was concerned about getting four useless success notifications, but now that I've switched to failure only, this should work.